src/commands/dev/dev.js

```diff
@@ -472,7 +495,19 @@ const dev = async (options, command) => {
     capabilities,
     timeouts,
   })
-  await startFrameworkServer({ settings })
+
+  log(`${NETLIFYDEVWARN} Setting up local development server`)
```

Comment:
Adding this as a progress message because installing build plugins might take a few seconds.